.github/workflows/ci.yml

```diff
@@ -12,7 +12,7 @@ jobs:
 
     php-tests:
         name: Tests - PHP ${{ matrix.php }} ${{ matrix.dependency-version }}
-        runs-on: ubuntu-latest
+        runs-on: ubuntu-22.04
```

Comment:
What do we gain again by fixing the version? We'll have to update this eventually 😬

Reply:
We gain control over when this happens, just like we control the major versions on the other action things.

Reply:
Right but actions can have BC breaks on major versions.

I don't see any way the Ubuntu version would break the CI?

Reply:
Ubuntu version typically does have breaking changes, and different actions may not be compatible yet.

Reply:
GitHub also doesn't upgrade everything at once, doing a rolling move per repo, so it's kinda janky to sometimes have one version, and sometimes another.